postgres-driver/error.go

```diff
@@ -2,14 +2,19 @@ package postgresdriver
 
 import (
 	"context"
+	"time"
 
 	"github.com/pokt-foundation/transaction-db/types"
 )
 
 func (d *PostgresDriver) WriteError(ctx context.Context, poktErr types.Error) error {
+	now := time.Now()
+
 	return d.InsertError(ctx, InsertErrorParams{
```

Comment:
Also need an `error_type` field to quickly know which kind of error we're looking to (i.e sync check error`). Such field could be an enum for now with the values of `sync_check`, `chain_check` and `relay_err` but more can be added later